testing/py-bleach/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: William Waisse <alpine@ww7.be>
+# Maintainer: William Waisse <alpine@ww7.be>
+pkgname=py-bleach
+_pkgname=bleach
+pkgver=3.0.2
+pkgrel=3
+pkgdesc="Bleach is an allowed-list-based HTML sanitizing library that escapes or strips markup and attributes."
+url="https://pypi.org/project/bleach/"
+arch="noarch"
+license="Apache-2.0"
+depends=""
+makedepends="python2-dev python3-dev py-setuptools py3-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+#source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+source="https://files.pythonhosted.org/packages/ae/31/680afc7d44040004296a2d8f0584983c2f2386448cd9d0964197e6c1160e/bleach-3.0.2.tar.gz"
```

Comment:
needs fix

Reply:
I have no idea about wich tests you could want :(  I fixed the non normalized source url )